content/reference-rippled.md

```diff
@@ -8624,8 +8654,14 @@ The `state` object may have some arrangement of the following fields:
 | load | Object | *Admin only* Detailed information about the current load state of the server |
 | load.job\_types | Array | *Admin only* Information about the rate of different types of jobs the server is doing and how much time it spends on each. |
 | load.threads | Number | *Admin only* The number of threads in the server's main job pool. |
-| load\_base | Number | This amount of server load is the baseline that is used to decide how much to charge in transaction fees; if the `load_factor` is equal to the `load_base` then only the base fee is enforced; if the `load_factor` is double the `load_base` then transaction fees are doubled. |
-| load\_factor | Number | The load factor the server is currently enforcing. The ratio between this value and the load\_base determines the multiplier for transaction fees. The load factor is determined by the highest of the individual server's load factor, cluster's load factor, and the overall network's load factor. |
+| load\_base | Integer | This amount of server load is the baseline that is used to decide how much to charge in transaction fees; if the `load_factor` is equal to the `load_base` then only the base fee is enforced; if the `load_factor` is double the `load_base` then transaction costs are doubled. |
```

Comment:
"This amount of server load is the baseline that is used to decide how much to charge in transaction fees; if the load_factor is equal to the load_base then only the base fee is enforced; if the load_factor is double the load_base then transaction costs are doubled."

Suggested rephrase: "The baseline used [by `rippled`?] to decide how much to charge in [transaction fees](content/concept-fees.html). If the `load_factor`is equal to the `load_base`, then only the base fee is enforced. If the `load_factor` is double the `load_base`, then [transaction costs](content/concept-transaction-cost.html) are doubled."

Also, is it worth explaining, here or somewhere else, what happens if the `load_factor` is 1.5 or 3 times the `load_base`? I think the answer is implied, but would probably be making that explicit since this is a tricky issue.